gradle.properties

```diff
@@ -18,4 +18,7 @@
 
 kotlin.code.style=official
 org.gradle.daemon=true
-org.gradle.jvmargs=-Xmx2560m
\ No newline at end of file
+org.gradle.jvmargs=-Xmx2560m
+
+kotlin.native.enableDependencyPropagation=false
```

Comment:
I'm quite curious about those two lines ^^

Reply:
it's for enable iosMain support with commonizer. 
https://kotlinlang.org/docs/reference/migrating-multiplatform-project-to-14.html#try-the-hierarchical-project-structure

Reply:
Thks!